content/chat/rooms/moderation/hive-model-only.textile

```diff
@@ -9,4 +9,29 @@ languages:
   - kotlin
 ---
 
-Hive documentation to go here.
+"Hive Moderation":https://hivemoderation.com is a powerful suite of moderation tools that can be used to moderate content in chat rooms.
+
+The Hive (Model Only) rule is a rule applied to chat rooms in Ably Chat which allows you to use Hive's text moderation models to detect and handle inappropriate content before it is published to other users.
+
+h2(#fields). Rule fields
+
+- Hive API Key := The API key for your Hive account.
```

Comment:
Do we need to capitalise every word in the header part of each bullet point? 
With the rest of our documentation I believe we tend to capitalise only the first letter of the first word (unless it's any other word is a name)

Reply:
Agreed - fixed